content/verification/veteran-confirmation/release-notes/2024-10-07.md

```diff
@@ -0,0 +1,10 @@
+Version v0 of the Veteran Confirmation API is deprecated.
+
+When we deprecate a version of an API, it will still be supported, but its use is discouraged. Deactivation has been extended to January 30, 2025. We’ll honor service-level agreements, but no new features will be added to the API version and only critical defects will be considered for correction.
```

Comment:
When you say "Deactivation has been extended to January 30, 2024," do you deactivation of the API? Or just the version of the API?

Reply:
Sorry its "deactivation of version v0 of the Veteran Confirmation API"